editors/code/package.json

```diff
@@ -311,27 +284,20 @@
         "sourcepawn.UploadOptions": {
           "type": "object",
           "default": {
-            "user": "",
-            "password": "",
             "host": "",
             "port": 21,
-            "localRoot": "",
-            "remoteRoot": "/public_html/remote-folder/",
-            "include": [
-              "**"
-            ],
+            "username": "",
+            "password": "",
+            "sftp": false,
+            "remoteRoot": "/tf/addons/sourcemod",
```

Comment:
Is this default intended?

Reply:
Yes, it actually is intended, opinionated assumption. Do you want another value there?
Also, I honestly don't know why I removed the `localRoot` variable and hardcoded it to the workspace root. Should I bring this back?

Reply:
In that case, `"/tf/addons/sourcemod"` is fine. I think workspace root is fine. The main point is to make it very hard for someone to accidentally delete their remote directory with this command.

Have you tested if there is some overwriting going on by default?

Reply:
Hm no. will do and report back.

Reply:
Thanks!

Reply:
If you were worried about files getting deleted from the remote location, if they were NOT present in the source, it's not happening in either of the FTP/SFTP libraries, which means it's not a directory synchronization. The only thing that happens is that existing files in destination are overwritten.